### DISCUSSION TITLE
Our changelog fragments are not compatible with markdown-lint

### DISCUSSION THREAD

**Discussion by ssbarnea** [ansible/ansible-navigator#709] [Linters, linting, formatting, pre-commit, CI]
Our markdown fragments do raise a linting error with markdown-lint: MD041 because they do not have a header. That is really annoying when writing them as using markdown-lint extension in vscode is quite popular and needed for our day to day work.

**Comment by webknjaz** [comment]
This is because of the nature of the fragments of not being full documents. There's nothing you can do on the fragments side, but this is why it's ignored in pre-commit. I suppose the ignore could be transferred to the linter's own config file if it exists (I haven't checked).
